src/sidebar/title.js

```diff
@@ -0,0 +1,24 @@
+import i18n from '@dhis2/d2-i18n'
+import { IconCross24, colors } from '@dhis2/ui'
+import PropTypes from 'prop-types'
+import React from 'react'
+import styles from './title.module.css'
+
+const Title = ({ onClose }) => (
+    <header className={styles.wrapper}>
+        <span className={styles.title}>{i18n.t('Details')}</span>
+        <button
+            className={styles.closeButton}
+            title={i18n.t('Close details sidebar')}
+            onClick={onClose}
+        >
+            <IconCross24 color={colors.grey700} />
+        </button>
+    </header>
+)
+
+Title.propTypes = {
+    onClose: PropTypes.func.isRequired,
```

Comment:
This is really a non-issue and I just want to write down what's on my mind right now..
I thought about the names of these callbacks recently. I think this should be called `onClick` or `close` (leaning towards the latter):

* When reading through the layout or sidebar component's code, `onClose` implies (to me) that this component reacts to when the sidebar is being closed (sth like: `onClose(() => doSth())`)
* `close` implies that this actually triggers some kind of action
* `onClick` leaves the decision about what should happen to the `Sidebar` (but one could argue that there's a cross icon in this component, so that separation does not make any sense, which is why I'm leaning towards the `close` option)
* `onCloseClick` is kind of the middle ground between the two above, inside the `<Layout />` component would do something like this: `sidebar={<Sidebar onCloseClick={hideSidebar} />`